onnx/backend/test.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+import numpy as np
+
+import onnx
+from onnx import helper
+from .test_util import create_input, N
+
+L = 20
+M = 10
+S = 5
+const2_np = np.random.randn(S, S)
+const2_onnx = onnx.helper.make_tensor("const2",
+                                      onnx.TensorProto.FLOAT,
+                                      (S, S),
+                                      const2_np.flatten().astype(float))
+
+# TODO: These Numpy specs will be generally useful to backend implementations,
```

Comment:
This TODO can be removed now :)

Reply:
Yeah I'm tempting to create a new place for hosting all these reference implementations instead of spreading them in a test file. :-)